recipes-core/packagegroups/packagefeed-ni-core.bb

```diff
@@ -30,5 +30,5 @@ RDEPENDS_${PN}_append_x64 = "\
 	rauc-mark-good \
 	nilrt-grub-runmode \
 	nilrt-grub-safemode \
-	${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'packagegroup-ni-xfce', '', d)} \
+	${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'packagegroup-ni-graphical', '', d)} \
```

Comment:
Since we only have one `DISTRO` these days and it always has the `x11` feature, we can drop the conditional logic here.